lib/tilelive.js

```diff
@@ -109,7 +109,7 @@ tilelive.auto = function(uri) {
     uri.protocol = uri.protocol || keyword + ':';
 
     if (!tilelive.protocols[uri.protocol]) {
-        [keyword, 'tilelive-' + keyword].forEach(function(name) {
+        [keyword, 'tilelive-' + keyword, '@mapbox/tilelive-' + keyword].forEach(function(name) {
```

Comment:
This is great @jingsam! The only thing I worry about is making the assumption that a tilelive module is under the `@mapbox` namespace. It definitely solves a majority of the modules, but there are others that could still hit the issue. Any ideas here?